docs/spec/lcd/readme.md

```diff
@@ -0,0 +1,47 @@
+# Overview
+
+These documents describe the Light Client Daemon, commonly referred to as LCD. The LCD is split into
+two separate components. 
+The first component is generic for any Tendermint based application. It 
+handles the security and connectivity aspects of following the header chain and verifying proofs
+from full nodes against locally trusted state roots. Furthermore it exposes exactly the same API as
+any Tendermint Core node.
+The second component is specific for the Cosmos Hub (Gaiad). It works through the query endpoint on
+Tendermint Core and exposes the application specific functionality, which can be arbitrary. All 
```

Comment:
Are these application specific proofs being verified in the LCD? This wording makes that unclear.

Reply:
I think they are now. @mossid worked on that.

Reply:
These are application specific.